lib/osf-components/addon/components/cedar-metadata-editor/component.ts

```diff
@@ -0,0 +1,21 @@
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+// import { task } from 'ember-concurrency';
+// import { tracked } from 'tracked-built-ins';
+import config from 'ember-osf-web/config/environment';
+
+const { cedarConfig } = config;
+
+interface Args {
+    instanceObj: any;
+    templateOjb: any;
```

Comment:
These `arg` names don't seem to match what's in the template file

Reply:
Lol, typo...